Cargo.toml

```diff
@@ -57,13 +57,13 @@ members = [
 ]
 exclude = [
   "fuzz",
-  "oak_enclave_shim",
+  "oak_enclave_binary",
```

Comment:
I would prefer removing the shim and just making the kernel a binary, or name this something like `oak_restricted_kernel_binary`.